_config.yml

```diff
@@ -61,10 +67,42 @@ pubs:
       abstract: A central challenge in the analysis of genetic variation is to provide realistic genome simulation across millions of samples. Present day coalescent simulations do not scale well, or use approximations that fail to capture important long-range linkage properties. Analysing the results of simulations also presents a substantial challenge, as current methods to store genealogies consume a great deal of space, are slow to parse and do not take advantage of shared structure in correlated trees. We solve these problems by introducing sparse trees and coalescence records as the key units of genealogical analysis. Using these tools, exact simulation of the coalescent with recombination for chromosome-sized regions over hundreds of thousands of samples is possible, and substantially faster than present-day approximate methods. We can also analyse the results orders of magnitude more quickly than with existing methods.
 
 videos:
-  - youtube-id: X1GEuQrF1jQ
-    title: Jerome Kelleher and Wilder Wohns give a detailed introduction to tree sequences and their inference. (2019)
+
+  - youtube-id: YrZTKjLzZY0
+    who: Wilder Wohns
+    year: 2020
+    where: Phyloseminar
+    title: Tree sequence fundamentals
+
   - youtube-id: yB4uSle66Pw
-    title: Yan Wong on tree-sequence inference. (2020)
+    who: Yan Wong
+    year: 2020
+    where: Phyloseminar
+    title: Tree sequences and inference
+
+  - youtube-id: yrDSLlssoek
+    who: Jerome Kelleher
+    title: Inferring the ancestry of everyone
+    where: PopGen Vienna
+    year: 2020
+
+  - youtube-id: X1GEuQrF1jQ?t=6
+    who: Wilder Wohns
+    where: MIA Primer
+    title: Introduction to the tree sequence toolchain
+    year: 2019
+
+  - youtube-id: X1GEuQrF1jQ?t=2825
```

Comment:
`?` needs to be `&` for the timestamp to work.

Reply:
Aha, great spot, thanks.